src/main/java/com/ajou/hertz/domain/user/controller/UserController.java

```diff
@@ -117,6 +119,24 @@ public ResponseEntity<UserResponse> signUpV1(
 			.body(UserResponse.from(userCreated));
 	}
 
+	@Operation(
+		summary = "프로필 이미지 변경",
+		description = "프로필 이미지를 변경합니다.",
+		security = @SecurityRequirement(name = "access-token")
+	)
+	@PutMapping(
+		value = "/me/profile-image",
+		headers = API_VERSION_HEADER_NAME + "=" + 1,
+		consumes = MediaType.MULTIPART_FORM_DATA_VALUE
+	)
+	public UserResponse updateProfileImageUrlV1(
```

Comment:
```suggestion
	public UserResponse updateProfileImageV1(
```

어차피 프로필 이미지 entity도 실제로 업데이트 할거라서 Url은 빼는 게 더 좋은 naming인 거 같아요.